package.json

```diff
@@ -8,9 +8,11 @@
   "sideEffects": false,
   "devDependencies": {
     "@types/jest": "^26.0.14",
+    "assert": "^2.0.0",
     "jest": "^26.4.2",
     "marked": "^0.4.0",
     "prettier": "^2.8.0",
+    "ts-expect": "^1.3.0",
```

Comment:
can you remove those changes from this PR please as the other PR also adds them?

also: why is `assert` needed? I don't see this used anywhere

Reply:
The `assert` is for `src/clone.test.ts`. 

As for removing the dependency on `ts-expect`, if I do the PR will fail its checks.